build3/premake4.lua

```diff
@@ -49,43 +49,55 @@
 
 --	--_OPTIONS["midi"] = "1";
 
-	newoption
-	{
-		trigger = "bullet2demos",
-		description = "Compile the Bullet 2 demos (Demo/Extra folder)"
-	}
```

Comment:
The **--bullet2demos** option was cruft: none of the other **premake4.lua** scripts were using it.